charts/finops-agent/values.yaml

```diff
@@ -102,18 +102,13 @@ gcpKey: ""
 ## @param logLevel The log level for the finops agent
 logLevel: "info"
 
-## export bucket configuration
-## @param exportBucket.config The config for the export bucket
+## federated storage configuration
+## @param federatedStorage.config The config for the federated storage
 ##
-exportBucket:
-  ## @param exportBucket.secret.create Create a secret for the export bucket
-  ## @param exportBucket.secret.config The config for the export bucket.
-  ## @param exportBucket.secret.existingSecret The name of an existing secret to use for the export bucket. Note, you cannot set both `create` and `existingSecret`.
+federatedStorage:
+  ## @param federatedStorage.config The config for the federated storage.
   ##
-  secret:
-    create: true
```

Comment:
I have removed this  parameter because the setting of the `federatedStorage.config` implies the intention to create a secret

Reply:
This won't prevent using a pre-existing config though right?